packages/frontend/src/hooks/fungibleTokensIncludingNEAR.js

```diff
@@ -11,5 +12,11 @@ export const useFungibleTokensIncludingNEAR = function () {
         selectTokensWithMetadataForAccountId(state, { accountId })
     );
 
-    return [NEARAsTokenWithMetadata, ...fungibleTokens];
+    const fungibleTokenPrices = useSelector(selectTokensFiatValueUSD);
+    const fungibleTokensWithPrices = fungibleTokens.map((ft) => ({
```

Comment:
**Not a blocker for this PR**, but this should be memoized so that we don't create an entirely new value for `fungibleTokensWithPrices` here _every time this function runs_.